### ISSUE TITLE
GraphQL `transform` directive doesn't respect `transformGifs` config setting

### ISSUE THREAD

**Issue by slavanossar** [craftcms/cms#6407]
### Description
When using the `transform` directive in a GraphQL query, transforms are still generated for GIFs even when `transformGifs: false` is set in `config/general.php`.

The directive uses [`Craft::$app->getAssets()->getAssetUrl()`](https://github.com/craftcms/cms/blob/2bbe993bbad4ab7472aa68086699a530307deac9/src/gql/directives/Transform.php#L115) which doesn't seem to check `transformGifs`, unlike the [`getUrl` method on `craft\elements\asset`](https://github.com/craftcms/cms/blob/2bbe993bbad4ab7472aa68086699a530307deac9/src/elements/Asset.php#L947).

### Steps to reproduce

1. Set `transformGifs: false` in `config/general.php`
2. Create an asset field and define an image transform.
3. Upload GIF to asset field.
4. Query asset and request transform, e.g.:

```gql
{
  asset(id: "1") {
    ... on assets_Asset {
      url
      resized: url @transform(handle: "myTransform")
    }
  }
}
```

5. Response

```js
// Expected
{
  "data": {
    "asset": {
      "url": "https://site.com/assets/image.gif",
      "resized": "https://site.com/assets/image.gif"
    }
  }
}

// Actual
{
  "data": {
    "asset": {
      "url": "https://site.com/assets/image.gif",
      "resized": "https://site.com/assets/_myTransform/image.gif"
    }
  }
}
```

### Additional info

- Craft version: 3.4.27
- PHP version: 7.1.32
- Database driver & version: Postgres 12.1
- Plugins & versions: N/A

**Comment by andris-sevcenko** [comment]
Just fixed this for the next 3.4 release.

Thanks for reporting!

**Comment by brandonkelly** [comment]
Craft 3.4.29 iso ut now with that fix.
